packages/page-staking/src/Actions/Pool/useAccountInfoManualRewards.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017-2022 @polkadot/app-staking authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Option } from '@polkadot/types';
+import type { PalletNominationPoolsPoolMember } from '@polkadot/types/lookup';
+import type { AccountInfo } from './types';
+
+import { useEffect, useState } from 'react';
+
+import usePoolInfo from '@polkadot/app-staking/Pools/usePoolInfo';
+import { createNamedHook, useApi, useCall, useIsMountedRef } from '@polkadot/react-hooks';
+import { PalletNominationPoolsBondedPoolInner, PalletNominationPoolsRewardPool } from '@polkadot/types/lookup';
+import { BN } from '@polkadot/util';
+
+const OPT_DEL = {
+  transform: (opt: Option<PalletNominationPoolsPoolMember>): PalletNominationPoolsPoolMember | null =>
+    opt.unwrapOr(null)
+};
+
+// as per polkadot-staking-dashboard code
+function calculatePayout (
+  bondedPool: PalletNominationPoolsBondedPoolInner,
+  rewardPool: PalletNominationPoolsRewardPool,
+  member: PalletNominationPoolsPoolMember,
+  rewardAccountBalance: BN
+) {
+  const rewardCounterBase = new BN(10).pow(new BN(18));
+
+  // convert needed values into BNs
+  const totalRewardsClaimed = new BN(rewardPool.totalRewardsClaimed);
+  const lastRecordedTotalPayouts = new BN(rewardPool.lastRecordedTotalPayouts);
+  const memberLastRecordedRewardCounter = new BN(member.lastRecordedRewardCounter);
+  const poolLastRecordedRewardCounter = new BN(rewardPool.lastRecordedRewardCounter);
+  const bondedPoolPoints = new BN(bondedPool.points);
+  const points = new BN(member.points);
+  const rewardPoolBalance = rewardAccountBalance;
```

Comment:
In the [original](https://github.com/Cardinal-Cryptography/polkadot-staking-dashboard/blob/master/src/contexts/Pools/ActivePools/index.tsx#L558-L562), this is
```javascript
// calculate the latest reward account balance minus the existential deposit
    const rewardPoolBalance = BN.max(
      new BN(0),
      new BN(rewardAccountBalance).sub(consts.existentialDeposit)
    );
```

Can you explain the change?

Reply:
That calculation ale already done in https://github.com/Cardinal-Cryptography/apps/blob/alephzero/packages/page-staking/src/Pools/usePoolInfo.ts#L51